### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by okpierre**
Updated readme to 
-include creation of dolphin user
-login as dolphin user to install

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,15 +113,19 @@ Please install and setup these softwares:
 
 ### Install Dolphin
 
-1. Clone the Dolphin repo
+1. Connect to misskey user
+
+	`su - misskey`
```
Comment (syuilo) [inline README.md]: dolphin?

Comment (okpierre) [reply README.md]: ooops yes - updated

**Comment by syuilo** [comment]
Thank you!

**Comment by okpierre** [comment]
> Thank you!

You're welcome!

**Pull request merged**